roles/pulp_devel/tasks/install_docs_requirements.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Cloning Pulpcore
+  git:
+    repo: 'https://github.com/pulp/pulpcore.git'
```

Comment:
I was expecting pulpcore would receive a non-source install in this case and that pulpcore would come from PyPI if unset. The reason is because a plugin developer who didn't configure their dev environment to use a source checkout from pulpcore should receive the GA bits, not a source checkout. Receiving a source checkout instead is a practical problem because plugins typically pin their version to <={whatever_pulpcore_on_master_is} so in practice this will fail in most cases.

Reply:
do you think we should disable https://github.com/pulp/pulp_installer/blob/master/roles/pulp_devel/tasks/install_docs_requirements.yml#L2-L21 when `pulp_source_dir` is not set? Because these tasks depend on `pulp_source_dir`

Reply:
When I grep https://github.com/pulp/pulp_installer/blob/master/roles/pulp_devel/tasks/install_docs_requirements.yml I don't see `pulp_source_dir`.

Reply:
comes from here: `pulp_requirements_dir: "{{ pulp_source_dir }}"`
https://github.com/pulp/pulp_installer/blob/master/roles/pulp_devel/defaults/main.yml

Reply:
Oh I see, thank you for explaining that. I'm ok with disabling those for now. The pulp_installer user didn't specify they wanted a pulpcore checkout so disabling https://github.com/pulp/pulp_installer/blob/master/roles/pulp_devel/tasks/install_docs_requirements.yml#L2-L21 seems reasonable.

A better way to fix this in the long term is for pulpcore to ship its dev/test/doc requirements outside of source installs. We have a ticket on the sprint to do just that [here](https://pulp.plan.io/issues/6844). After 6844 is implemented we'll have to wait for the 3.5 release to re-enable these tasks, so filing a future pulp_installer story to re-enable them I think would be good.

What do you think about this plan?

Reply:
sounds good to me